raster-foundry/raster-foundry
# Handle non-0 NODATA values

## Overview
Handle ingested scenes with implicit 0 NODATA value

### Checklist

- ~Styleguide updated, if necessary~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~

### Notes


### TODO
- [x]  update gt-contrib once it's released
- [ ] Write migration to add nodata value overrides to all existing RF ingested scenes
- [ ] Update ingest process to set the nodata value correctly when converting stuff to cogs
- [ ] Update the dev database with nodata values
- [ ] Run the nodata value migration on staging / prod

### Notes
Depending on the default nodata handling behavior, we might not need to do the migration stuff above

## Testing Instructions

- download the latest dev database and verify that scenes do not have black borders instead of valid transparent nodata
- On the landsat test scene, set the nodata value to something else in the scene details modal
- verify that black borders show up
- set it to 0 and verify that the black borders disappear
- Verify that nodata values are respected in single band color modes

Closes #5098
